cmd/acb/commands/exec/exec.go

```diff
@@ -186,7 +191,13 @@ var Command = cli.Command{
 			}
 		}
 
-		rendered, err := templating.LoadAndRenderSteps(template, renderOpts)
+		secretResolver, err := templating.DefaultSecretResolver(azureEnvironmentName)
+		if err != nil {
+			return err
+		}
+
+		// TODO: Do we need to add any timeout with the ctx for resolving the secrets?
```

Comment:
Yes. Better to have a timeout in case it's stuck somewhere. Otherwise users won't see any output from stream logs until the client side timeout.